playbooks/setup_forklift.yml

```diff
@@ -41,3 +41,13 @@
         # it can be removed when we use ansible-lint 4.3.0 as that contains a fix
         content: "{{ forklift_config | to_json | from_json | to_nice_yaml }}"
         dest: "{{ forklift_dest }}/vagrant/settings.yaml"
+
+    - name: 'Install Forklift collection dependencies'
+      command:
+        cmd: ansible-galaxy collection install -r requirements.yml
+        chdir: "{{ forklift_dest }}"
+
+    - name: 'Install Forklift pulp_installer role dependencies'
+      command:
+        cmd: ansible-galaxy role install -r playbooks/galaxy_collections/ansible_collections/pulp/pulp_installer/requirements.yml
+        chdir: "{{ forklift_dest }}"
```

Comment:
Ought this be done every time? Perhaps a variable would be the way to go here so th epipeline can set it?

Reply:
you mean putting this in a block/role and add a `when`? yeah, sure, why not.